packages/core/src/components/Label/__stories__/label.stories.tsx

```diff
@@ -18,6 +21,21 @@ export default {
   decorators: metaSettings.decorators
 };
 
+const withGrid: Decorator = Story => (
+  <div
```

Comment:
That's great! We want to extract these to storybook layout components and use them all over